schema.sql

```diff
@@ -362,7 +363,7 @@ CREATE TABLE IF NOT EXISTS `towns` (
   UNIQUE KEY `name` (`name`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
 
-INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '32'), ('players_record', '0');
+INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '32'), ('players_record', '0'), ('loyalty_updated', '0');
```

Comment:
```suggestion
INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '33'), ('players_record', '0'), ('loyalty_updated', '0');
```

I think you forgot to bump the schema version here.

Reply:
thanks! I'll fix it in the next commit, I'm still waiting more reviews